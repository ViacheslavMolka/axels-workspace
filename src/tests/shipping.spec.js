import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import ShippingComponent from '../components/shipping';


configure({ adapter: new Adapter() });

it('ShippingComponent snapshot', () => {
    const component = shallow(<ShippingComponent />);
    expect(component).toMatchSnapshot();
});
