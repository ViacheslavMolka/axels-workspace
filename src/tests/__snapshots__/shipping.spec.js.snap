// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingComponent snapshot 1`] = `ShallowWrapper {}`;
